__utils__/permissions.ts

```diff
@@ -16,7 +16,12 @@ export const Base: RawRuleOf<MongoAbility<Abilities>>[] = [
   },
   { action: "update", subject: "FormRecord", fields: ["isPublished"], inverted: true },
   {
-    action: ["create", "view", "update"],
+    action: ["view"],
```

Comment:
Removed "create" as no user should be able to create a user object as it is an internal system function.